docs/features/common_functional_options.md

```diff
@@ -165,3 +165,41 @@ The above example is updating the predefined command of the image, **appending**
 
 !!!info
     This can't be used to replace the command, only to append options.
+
+#### NewTerminateOptions
```

Comment:
I'm sorry but this is not properly documented on our docs 😢 

This file is for the functional options that affect the `Run` function in the modules API.

I'd add this here: https://golang.testcontainers.org/features/garbage_collector/#terminate-function if possible.

Reply:
no worries, i was almost certain i added it in the writing place 😅, will do update it thank you